pom.xml

```diff
@@ -10,7 +10,7 @@
   <groupId>com.auth0</groupId>
   <artifactId>jwks-rsa</artifactId>
   <packaging>jar</packaging>
-  <version>0.2.1-SNAPSHOT</version>
+  <version>0.2.2-SNAPSHOT</version>
```

Comment:
The version will be bumped when the release is prepared. Please rollback this change.